### PR TITLE
Complete Sort Functionality

### PR DESCRIPTION
**Pull request by aleksandar-cakic**
- Added CSS to match business document 
- Updated Helpfulness shared component to re-render on selected Sort filter (for reviews)

### CODE REVIEW

File: client/src/components/shared/Helpfulness.jsx [hunk 1]
```diff
@@ -47,6 +47,23 @@ class Helpfulness extends React.Component {
...
+      if (this.state.targetId !== targetId) {
+
+        this.setState({
+          helpfulCount: targetObj['helpfulness'],
+          targetId: targetId,
+          link: '/reviews/' + targetId + '/helpful'
+        });
+      }
+    }
+  }
```
Comment (VinhH2402) [inline client/src/components/shared/Helpfulness.jsx:65]: Hi Alex , just wonder why we need componentDidUpdate ? I have tested it and it work fine, are there any bugs ?

**Review by VinhH2402 — APPROVED** [review]
ready to merge

File: client/src/components/shared/Helpfulness.jsx (hunk 1 above)
Comment (aleksandar-cakic) [reply client/src/components/shared/Helpfulness.jsx:65]: When I'm using Sort button to sort my reviews, the helpfulness doesn't update. Let me know if you have a minute to test it out, maybe I'm missing something.

Comment (aleksandar-cakic) [reply client/src/components/shared/Helpfulness.jsx:65]: Per our chat, keeping ComponentDidUpdate for now, for Sort filters

**Pull request merged**